ui/components/app/custom-spending-cap/__snapshots__/custom-spending-cap.test.js.snap

```diff
@@ -24,7 +24,7 @@ exports[`CustomSpendingCap should match snapshot 1`] = `
                 class="mm-box form-field__heading-title mm-box--display-flex mm-box--align-items-baseline"
               >
                 <h6
-                  class="box mm-text mm-text--body-sm-bold box--display-inline-block box--flex-direction-row box--color-text-default"
+                  class="mm-box mm-text mm-text--body-sm-bold mm-box--display-inline-block mm-box--color-text-default"
                   for="custom-spending-cap"
                   tag="label"
                 >
```

Comment:
Removes unused default from old Box component `box--flex-direction-row` the display of this component is inline-block. Other styles are the same